samples/bluetooth/hci_lpuart/boards/thingy91_nrf52840.conf

```diff
@@ -0,0 +1,24 @@
+# Override prj.conf defaults
+
+CONFIG_GPIO=y
+CONFIG_BT_CTLR=y
+CONFIG_MAIN_STACK_SIZE=1024
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=512
+CONFIG_BT_WAIT_NOP=y
+CONFIG_BT_MAX_CONN=16
+CONFIG_BT_TINYCRYPT_ECC=n
```

Comment:
Why is this needed ?
Seems to have dependencies to bt stack causing it to be unavailable and thus `n` per default.

Reply:
Inherited from the nRF9160DK configuration, but I've removed it.